### PR TITLE
[DDW-551] Adjust sorting on rewards screen

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR adjusted sorting of table on rewards screen
[Jira Ticket](https://jira.iohk.io/browse/DDW-551)

## Todos
- [x] Adjust sorting of table on rewards screen

## Testing Checklist
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1612300182137000)

### English
image
image

### Japanese
image
image

## Test Cases

```Gherkin
Scenario 1: Sort reward
Given I am on Staking -> Rewards screen
And there are at least 2 or more wallets displayed
And at least 2 wallets have different rewards amount
When I sort the rewards in (ascending|descending) order
Then they are sorted correctly
```

### App
- [x] Go to `Staking` => `Rewards` and make sure sorting of table works properly whenever changing `column` and `order`

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/utils/sortComparators.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  if (stringA > stringB) {
+    return isAscending ? 1 : -1;
+  }
+
+  return 0;
+};
+
+export const dateComparator = (
```
Comment (nikolaglumac) [inline source/renderer/app/utils/sortComparators.js:37]: @yakovkaravelov please add return type.

File: source/renderer/app/utils/sortComparators.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  if (numberA.isGreaterThan(numberB)) {
+    return isAscending ? 1 : -1;
+  }
+
+  return 0;
+};
+
+export const stringComparator = (
```
Comment (nikolaglumac) [inline source/renderer/app/utils/sortComparators.js:21]: @yakovkaravelov please add return type.

File: source/renderer/app/utils/sortComparators.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+// @flow
+import BigNumber from 'bignumber.js';
+import moment from 'moment';
+
+export const bigNumberComparator = (
```
Comment (nikolaglumac) [inline source/renderer/app/utils/sortComparators.js:5]: @yakovkaravelov please add return type.

**Review by nikolaglumac — CHANGES_REQUESTED** [review]
Please fix type declarations 🙏 
The rest looks good.

Don't forget to add screenshots!

**Comment by yakovkaravelov** [comment]
@nikolaglumac Fixed.
@mioriohk @gnpf This is ready for test, thanks.

**Review by nikolaglumac — APPROVED** [review]
Looks good 👍

**Review by mioriohk — APPROVED** [review]
LGTM 👍 . Tested on 16224

**Pull request merged**